cpp/benchmarks/io/parquet/parquet_reader_multithread.cpp

```diff
@@ -50,7 +50,7 @@ std::string get_label(std::string const& test_name, nvbench::state const& state)
 }
 
 std::tuple<std::vector<cuio_source_sink_pair>, size_t, size_t> write_file_data(
-  nvbench::state& state, std::vector<cudf::type_id> const& d_types)
+  nvbench::state& state, std::vector<cudf::type_id> const& d_types, io_type const io_source_type)
```

Comment:
```suggestion
  nvbench::state& state, std::vector<cudf::type_id> const& d_types, io_type io_source_type)
```
There are [many discussions](https://stackoverflow.com/questions/117293/does-using-const-on-function-parameters-have-any-effect-why-does-it-not-affect) about it but passing const value is not encouraged in general. Not a hard requirement though.

Reply:
Makes sense. Let me remove it.